vpr/src/place/median_move_generator.h

```diff
@@ -0,0 +1,9 @@
+#ifndef VPR_MEDIAN_MOVE_GEN_H
+#define VPR_MEDIAN_MOVE_GEN_H
+#include "move_generator.h"
+
+class MedianMoveGenerator : public MoveGenerator {
```

Comment:
Comment exactly what this does (how does the move work and why)

Reply:
Done.